index.js

```diff
@@ -107,8 +107,16 @@ class Conf {
 			let data = fs.readFileSync(this.path, this.encryptionKey ? null : 'utf8');
 
 			if (this.encryptionKey) {
-				const decipher = crypto.createDecipher('aes-256-cbc', this.encryptionKey);
-				data = Buffer.concat([decipher.update(data), decipher.final()]);
+				try {
+					return Object.assign(obj(), JSON.parse(data));
```

Comment:
Wouldn't it make more sense to try to decrypt first and rather fall back to `Object.assign(obj(), JSON.parse(data));` if decryption fails? Your way here adds a slight overhead each time instead of just the first.

Reply:
Good point, I think I was over-thinking the whole thing actually, since that was my first approach!